code_samples/map_removeAll_sample.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
Can you add copyright? See other files

Reply:
I would add it but could not be sure should I copy/paste the copyright or is there a way to do it?

Reply:
Just copy paste